packages/node_modules/@webex/plugin-meetings/src/reconnection-manager/index.js

```diff
@@ -205,9 +185,10 @@ export default class ReconnectionManager {
    * @private
    * @memberof ReconnectionManager
    */
-  static cleanup(meeting) {
-    // close pcs, unset to null and reuse the streams with new pcs
+  static setupPeerconnection(meeting) {
```

Comment:
Minor but we use `PeerConnection` everywhere else in this file, so let's use that capitalization.